addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
 
     const {
       tagName,
       ignoreTouchAction,
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
```

Comment:
Also, long term, because this mixin is universally applied, I want to give user's an opt out for it and I want to wrap it in a perf test to see what kind of an affect it has on apps.  This will get easier soon as I'm closing to landing a stable release of heimdalljs.